app/helpers/provisioning_templates_helper.rb

```diff
@@ -13,7 +13,7 @@ def template_kind(template)
   end
 
   def include_javascript
-    javascript 'provisioning_template', 'ace/ace',
+    javascript 'editor', 'ace/ace',
```

Comment:
Perhaps he was expecting this method to have been removed, given the one above - however the one above is only in common_parameters_helper.  I'd suggest moving it to one of the app-wide helpers or a new EditorHelper (renaming it to something less generic in the process), then delete the duplicates in these two.